plugins/source/aws/resources/services/resiliencehub/recommendation_templates.go

```diff
@@ -0,0 +1,30 @@
+package resiliencehub
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/resiliencehub/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func recommendationTemplates() *schema.Table {
+	return &schema.Table{
+		Name:        "aws_resiliencehub_recommendation_templates",
+		Description: `https://docs.aws.amazon.com/resilience-hub/latest/APIReference/API_RecommendationTemplate.html`,
+		Resolver:    fetchRecommendationTemplates,
+		Transform:   transformers.TransformWithStruct(&types.RecommendationTemplate{}, transformers.WithPrimaryKeys("Name")),
```

Comment:
`Name` is the only PK? Should add `account_id` (and maybe `region` too) to avoid PK issues with multiple accounts unless Name is required to be unique globally.